srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 3 nodes, setting nnodes to 1
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n08: task 0: Exited with exit code 1
srun: Terminating job step 9433433.0
srun: error: sh02-01n08: task 0: Exited with exit code 1
srun: Terminating job step 9433433.7
2160
2160
2160
2160
2160
srun: error: sh02-01n08: task 0: Exited with exit code 1
srun: error: sh02-01n08: task 0: Exited with exit code 1
srun: error: sh02-01n08: task 0: Exited with exit code 1
srun: Terminating job step 9433433.2
srun: Terminating job step 9433433.5
srun: error: sh02-01n08: task 0: Exited with exit code 1
srun: Terminating job step 9433433.8
srun: error: sh02-01n08: task 0: Exited with exit code 1
srun: error: sh02-01n08: task 0: Exited with exit code 1
2160
srun: error: sh02-01n08: task 0: Exited with exit code 1
2160
srun: Terminating job step 9433433.4
srun: Terminating job step 9433433.6
srun: Terminating job step 9433433.3
srun: Terminating job step 9433433.1
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433433.25
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433433.28
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433433.26
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n10: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: Terminating job step 9433433.27
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: error: sh02-01n10: task 0: Exited with exit code 1
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: Terminating job step 9433433.29
srun: Terminating job step 9433433.20
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433433.22
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433433.23
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433433.24
srun: Terminating job step 9433433.21
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433433.19
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433433.16
srun: Terminating job step 9433433.9
srun: Terminating job step 9433433.15
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433433.17
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433433.18
srun: Terminating job step 9433433.14
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433433.10
srun: Terminating job step 9433433.11
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433433.12
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433433.13
